test/index.js

```diff
@@ -92,15 +92,6 @@ test('if add will return the value= if an empty key', (t) => {
   t.is(result, 'value');
 });
 
-test('if add will add the value to the array if an empty key', (t) => {
```

Comment:
this is not a duplicate ... the first method tests how a `null` key is handled for `add` on an *object*, whereas the second method tests how a `null` key is handled for `add` on an *array*. If you look at the bodies of each test you'll see the `object` and `result` are quite different.

That said, the name of the tests could be more specific to prevent confusion. I'll update this.

Reply:
Oh, you're right. Just the words are the same...

    const object = ['top', 'level'];
    const object = {top: 'level'};

I'm sorry for the trouble.

Reply:
Actually, I double-checked the tests and I think, that you explained the difference between tests on lines 19 and 86. But how about the tests on lines 19 and 95? Their bodies are 100% the same and they use only local variables.

https://github.com/planttheidea/unchanged/blob/master/test/index.js#L19

    test('if add will add the top-level value to the array', (t) => {
      const object = ['top', 'level'];

      const result = index.add(null, 'value', object);

      t.not(result, object);
      t.deepEqual(result, [...object, 'value']);
    });

https://github.com/planttheidea/unchanged/blob/master/test/index.js#L86

    test('if add will return the value itself if an empty key is used for "add" to an array', (t) => {
      const object = {top: 'level'};

      const result = index.add(null, 'value', object);

      t.not(result, object);
      t.is(result, 'value');
    });

https://github.com/planttheidea/unchanged/blob/master/test/index.js#L95

    test('if add will append the value to the array if an empty key is used for "add" on an array', (t) => {
      const object = ['top', 'level'];

      const result = index.add(null, 'value', object);

      t.not(result, object);
      t.deepEqual(result, [...object, 'value']);
    });

I don't think, that you'll find a naming convention, which makes those two ones test something else ;-)

Reply:
Ah yes, you are correct here, tests at lines 19 and 95 are indeed duplicates. If you can create a new PR for the removal of L19 (the less-explicit duplicate) I will happily merge that. And no worries about the original callout, that verbiage was pretty vague so its understandable how it was seen that way. :)

Reply:
Done. Have a look at #8, please.